text/0041-extensible-block-body.md

```diff
@@ -0,0 +1,104 @@
+# Extensible Block Body
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0041](https://github.com/casperlabs/ceps/pull/0041)
+
+Modify the internal structure of the block body, so that it will be easy to add new fields to it in the future in a backwards-compatible manner.
```

Comment:
Is it really about modifying the internal structure of the block body or only about how its hash is computed?

Reply:
It's also kind of about modifying how the block body is stored. But you're right, this is phrased badly.